cql3/statements/select_statement.cc

```diff
@@ -831,6 +831,16 @@ indexed_table_select_statement::prepare(database& db,
     const auto& im = index_opt->metadata();
     sstring index_table_name = im.name() + "_index";
     schema_ptr view_schema = db.find_schema(schema->ks_name(), index_table_name);
+    std::optional<global_index_prepare_data> global_index_data;
+
+    if (!im.local()) {
```

Comment:
We might have separate `*_indexed_table_select_statement` classes for local and global indexes, if their execution is different enough to require different data structures. Not proposing to do it now.